t/templates/wrapper.tt

```diff
@@ -3,11 +3,23 @@
 <head>
     <title>[% title %]</title>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link href="http://netdna.bootstrapcdn.com/bootstrap/3.0.3/css/bootstrap.min.css" rel="stylesheet">
-    <link href="atom.xml" rel="alternate" title="[% plerd.title %]" type="application/atom+xml">
+    <link href="https://netdna.bootstrapcdn.com/bootstrap/3.0.3/css/bootstrap.min.css" rel="stylesheet">
```

Comment:
Why add all this to the t/ templates now?

(It may be a good idea, but it's unrelated to the tag work...)

Reply:
Oh, it looks like you pasted in wrapper.tt from templates/wrapper.tt.

I don't think that's at all necessary just to make the tests work, right? Go ahead and just revert this file, please.